index.yaml

```diff
@@ -2637,6 +2637,36 @@ entries:
     - assets/rancher-alerting-drivers/rancher-alerting-drivers-1.0.100.tgz
     version: 1.0.100
   rancher-backup:
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-backup-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Rancher Backups
+      catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
+      catalog.cattle.io/namespace: cattle-resources-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
+      catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
```

Comment:
This annotation should be updated to 
catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'